packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, lookup, owner;
-var originalLookup = Ember.lookup;
-
-QUnit.module('ember-htmlbars: {{#unbound}} helper', {
-  setup() {
-    Ember.lookup = lookup = { Ember: Ember };
-
-    view = EmberView.create({
-      [OWNER]: owner,
-      template: compile('{{unbound foo}} {{unbound bar}}'),
-      context: EmberObject.create({
-        foo: 'BORK',
-        barBinding: 'foo'
-      })
-    });
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('it should render the current value of a property on the context', function() {
-  equal(view.$().text(), 'BORK BORK', 'should render the current value of a property');
-});
-
-QUnit.test('it should not re-render if the property changes', function() {
-  run(function() {
-    view.set('context.foo', 'OOF');
-  });
-  equal(view.$().text(), 'BORK BORK', 'should not re-render if the property changes');
-});
-
-QUnit.test('it should not re-render if the parent view rerenders', function() {
-  run(function() {
-    view.set('context.foo', 'OOF');
-    view.rerender();
-  });
-  equal(view.$().text(), 'OOF OOF', 'should re-render if the parent view rerenders');
-});
-
-QUnit.test('it should assert unbound cannot be called with multiple arguments', function() {
-  expectAssertion(function() {
-    runAppend(EmberView.create({
-      template: compile('{{unbound foo bar}}'),
-      context: EmberObject.create({
-        foo: 'BORK',
-        bar: 'foo'
-      })
-    }));
-  }, /unbound helper cannot be called with multiple params or hash params/);
-});
-
-QUnit.test('should property escape unsafe hrefs', function() {
-  /* jshint scripturl:true */
-
-  expect(3);
-
-  runDestroy(view);
-
-  view = EmberView.create({
-    template: compile('<ul>{{#each view.people as |person|}}<a href="{{unbound person.url}}">{{person.name}}</a>{{/each}}</ul>'),
-    people: emberA([{
-      name: 'Bob',
-      url: 'javascript:bob-is-cool'
-    }, {
-      name: 'James',
-      url: 'vbscript:james-is-cool'
-    }, {
-      name: 'Richard',
-      url: 'javascript:richard-is-cool'
-    }])
-  });
-
-  runAppend(view);
-
-  var links = view.$('a');
-  for (var i = 0, l = links.length; i < l; i++) {
-    var link = links[i];
-    equal(link.protocol, 'unsafe:', 'properly escaped');
-  }
-});
-
-QUnit.test('should render on attributes', function(assert) {
-  /* jshint scripturl:true */
-
-  runDestroy(view);
-
-  view = EmberComponent.create({
-    layout: compile('<a href={{unbound name}}></a>'),
-    name: 'bob'
-  });
-
-  runAppend(view);
-
-  run(function() {
-    view.set('name', 'rick');
-  });
-  assert.equal(view.$().html(), '<a href="bob"></a>');
-});
-
-QUnit.module('ember-htmlbars: {{#unbound}} helper with container present', {
```

Comment:
@rwjblue we both don't understand what these tests are supposed to be testing, can you halp?